.circleci/config.yml

```diff
@@ -20,6 +20,13 @@ var_4: &init_environment
       # Setup circleci environment
       ./.circleci/env.sh
 
+      # Install GTK+ graphical user interface (libgtk-3-0), advanced linux sound architecture (libasound2)
+      # and network security service libraries (libnss3) & X11 Screen Saver extension library (libssx1)
+      # which are dependendies of chrome & needed for karma & protractor headless chrome tests.
+      # This is a very small install with the whole init_environment step taking less than 8 seconds.
```

Comment:
todo for using headless_shell to get away from this requirement?
will be a nuisance for our users too